contracts/src/v0.8/functions/dev/1_0_0/HasRouter.sol

```diff
@@ -42,11 +44,19 @@ abstract contract Routable is ITypeAndVersion, IConfigurable {
   function _updateConfig(bytes memory config) internal virtual;
 
   /**
-   * @inheritdoc IConfigurable
+   * @dev Internal function that can be re-used in the constructor
    */
-  function updateConfig(bytes memory config) external override onlyRouter {
+  function _handleConfigUpdate(bytes memory config) private {
```

Comment:
nit: please don't place functions between the public and internal (_)updateConfig